cognite/client/data_classes/cdm/v1.py

```diff
@@ -12,103 +12,38 @@
-    source_created_time = PropertyOptions("sourceCreatedTime")
-    source_updated_time = PropertyOptions("sourceUpdatedTime")
-    source_created_user = PropertyOptions("sourceCreatedUser")
-    source_updated_user = PropertyOptions("sourceUpdatedUser")
 
+class CogniteDescribableProperties:
     @classmethod
     def get_source(cls) -> ViewId:
-        return ViewId("cdf_cdm_experimental", "CogniteSourceable", "v1")
-
-
-class CogniteSourceableNodeApply(CogniteSourceableProperties, TypedNodeApply):
-    def __init__(
-        self,
-        space: str,
-        external_id: str,
-        source_id: str | None = None,
-        source_context: str | None = None,
-        source: DirectRelationReference | tuple[str, str] | None = None,
-        source_created_time: datetime | None = None,
-        source_updated_time: datetime | None = None,
-        source_created_user: str | None = None,
-        source_updated_user: str | None = None,
-        existing_version: int | None = None,
-        type: DirectRelationReference | tuple[str, str] | None = None,
-    ) -> None:
-        TypedNodeApply.__init__(self, space, external_id, existing_version, None, type)
-        self.source_id = source_id
-        self.source_context = source_context
-        self.source = DirectRelationReference.load(source) if source else None
-        self.source_created_time = source_created_time
-        self.source_updated_time = source_updated_time
-        self.source_created_user = source_created_user
-        self.source_updated_user = source_updated_user
+        return ViewId("cdf_cdm_experimental", "CogniteDescribable", "v1")
 
 
-class CogniteSourceableNode(CogniteSourceableProperties, TypedNode):
-    def __init__(
-        self,
-        space: str,
-        external_id: str,
-        version: int,
-        last_updated_time: int,
-        created_time: int,
-        source_id: str | None = None,
-        source_context: str | None = None,
-        source: DirectRelationReference | None = None,
-        source_created_time: datetime | None = None,
-        source_updated_time: datetime | None = None,
-        source_created_user: str | None = None,
-        source_updated_user: str | None = None,
-        type: DirectRelationReference | None = None,
-        deleted_time: int | None = None,
-    ) -> None:
-        TypedNode.__init__(self, space, external_id, version, last_updated_time, created_time, deleted_time, None, type)
-        self.source_id = source_id
-        self.source_context = source_context
-        self.source = DirectRelationReference.load(source) if source else None
-        self.source_created_time = source_created_time
-        self.source_updated_time = source_updated_time
-        self.source_created_user = source_created_user
-        self.source_updated_user = source_updated_user
+class CogniteDescribableNodeApply(CogniteDescribableProperties, TypedNodeApply):
+    """This represents the writing format of cognite describable node.
```

Comment:
Quite a lot of lower-cased `cognite`:
```suggestion
class CogniteDescribableNodeApply(CogniteDescribableProperties, TypedNodeApply):
    """This represents the writing format of Cognite describable node.
```